src/components/PrivacyStatement/index.js

```diff
@@ -0,0 +1,54 @@
+import { h, Component} from 'preact'
+
+import theme from '../Theme/style.css'
+import style from './style.css'
+import Title from '../Title'
+import {preventDefaultOnClick} from '../utils'
+import {sendScreen} from '../../Tracker'
+import {parseStringWithHTMLTags} from '../../locales'
+
+const externalUrls = {
+  terms: process.env.ONFIDO_TERMS_URL,
+  privacy: process.env.ONFIDO_PRIVACY_URL
+}
+
+class PrivacyStatement extends Component {
+  componentDidMount() {
+    sendScreen(['privacy'])
+  }
+
+  render({i18n, back, acceptTerms}) {
+    const title = i18n.t('privacy.title')
+    return (
+      <div className={style.privacy}>
+        <Title {...{title}} />
+        <div className={`${theme.thickWrapper} ${style.content}`}>
+          <ul className={style.list}>
+            <li className={style.item}>{i18n.t('privacy.item_1')}</li>
+            <li className={style.item}>{i18n.t('privacy.item_2')}</li>
+            <li className={style.item}>{i18n.t('privacy.item_3')}</li>
+          </ul>
+
+          <div>
+            <div className={style.smallPrint}>
+              {parseStringWithHTMLTags(i18n.t('privacy.small_print')).map(
```

Comment:
Nice :) Just a couple of comments on this:

1. I would personally have this wrap the `i18n.t` call itself, to make it 100% clear that we're just trusting the `i18n` source (rather than just randomly converting strings into HTML tags, which is dangerous)
2. This isn't parsing strings with HTML, it's parsing strings with XML
3. I would change the api of this function so that the `.map` part is implicit, rather than having to manually do it each time
4. I would change the api of this function so the case of `typeof value === "string"` case is implicit in returning that string, rather than manually doing so each time

e.g.
```javascript
parseI18nWithXmlTags('privacy.small_print', tagElement => (
  <a href={externalUrls[tagElement.type]} target='_blank'>{tagElement.text}</a>
));
```